q2_keegan/assets/scooper/index.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block head %}
+<style type="text/css">
+  .viz {
+    padding-top: 55px;
+  }
+
+</style>
+{% endblock %}
```

Comment:
Is this block necessary? It doesn't look like you're applying this CSS class anywhere else in this viz, so it looks like it might be unused code, to me. If it is unused, please remove.